packages/config/src/chains/astar.ts

```diff
@@ -12,6 +12,7 @@ export const shibuya = new Chain({
   assets: shibuyaAssets,
   defaultAsset: sbyAsset,
   endpoint: 'ws://127.0.0.1:9948',
+  network: 'rococo',
```

Comment:
It looks like that this `network` is specifically for the `options` of `getDerivativeAccount`?

Calling it `network` is very confusing since there’s already a `relayChain` variable.

Reply:
The values for the network are defined here:

https://github.com/paritytech/polkadot/blob/52209dcfe546ff39cc031b92d64e787e7e8264d4/xcm/src/v3/junction.rs#L52

Polkadot, Kusama, Westend, Rococo, ...

Incorrect values can lead to program errors.

The relayChain variable was previously used to identify the relay chain. It can be filled with strings like "local" or "moonbase-alpha-relay," for example.

Reply:
Yes, understood. I meant that this is an XCM specific config at the second level, but not a first level chain config. We should wrap up all the xcm parameters, such as.
```
key: 'shibuya',
assets: shibuyaAssets,
defaultAsset: sbyAsset,
endpoint: 'ws://127.0.0.1:9948',
xcm: { network, refTimePerSecond, proofSizePerMb, defaultInstructionCount } 
```